README.md

```diff
@@ -26,10 +26,24 @@ Asks the user for a directory.
 # platform details
 * OSX: uses Cocoa's NSAlert/NSSavePanel/NSOpenPanel classes
 * Win32: uses MessageBox/GetOpenFileName/GetSaveFileName (via package github.com/AllenDang/w32)
-* Linux: uses Gtk's MessageDialog/FileChooserDialog (via package github.com/mattn/gtk)
+* Linux: uses Gtk3's MessageDialog/FileChooserDialog (via package github.com/gotk3/gotk3)
 
 # build
+
+You will first need to set up go modules for you project. That can be done by running:
```

Comment:
I'll admit to being somewhat out of date with go development practices, but afaik modules remain an opt-in feature and as a basic component library I don't think dialog has any place telling users how to setup their projects?

Reply:
While I do see you point I also think that this change makes it much more clear as I was quite unsure about what to do when I initially tried to use this library. Users that know modules can just ignore it but for those who are less used to it, it makes a lot of sense in my opinion.